README.md

```diff
@@ -275,7 +259,7 @@ Allowed values are `traditional`, `syslog`, and `modern`. Default to `modern`.
 
 **Note:** Unless the above options are given, these local file outputs are configured.
 
-```
+```bash
```

Comment:
@richm markdownlint requires some syntax language to be specified with code blocks. Can you think of something other than bash in this case? The list of supported languages: https://docs.readme.com/rdmd/docs/code-blocks

Reply:
syntax language is required?  The original markdown spec says you can have fenced code blocks without specifying the syntax - https://www.markdownguide.org/extended-syntax/#fenced-code-blocks